packages/gestalt/src/ComboBox.jsdom.test.js

```diff
@@ -114,6 +114,8 @@ describe('ComboBox', () => {
         selectedOption={selectedOption}
         size={size}
         tags={tags}
+        hideWhenReferenceHidden={false} // Disabling as Popover uses IntersectionObserver which is not supported by testing-library
+        __disableFocusTrap // Disabling for testing purposes as focus trap makes other elements inaccessible
```

Comment:
This is the comment that should be where the prop is defined. It can be included here as well if you like, but the prop definition is more central